docs/modules/ROOT/pages/s3/how-to.adoc

```diff
@@ -1,8 +1,15 @@
-= How-To Create a Bucket
+= Create an S3 Bucket
```

Comment:
Let's call it an "Object Storage Bucket".

We should mention though, that it's S3 compatible.